### PR TITLE
fix some issue 

### PR DESCRIPTION
**Pull request by L1cardo**
1. add `toml` support #4 
2. add `properties` support
3. fix white flick

### CODE REVIEW

File: Sources/CodeMirror-SwiftUI/Representables/CodeView.swift [hunk 1]
```diff
@@ -124,8 +124,10 @@ extension CodeView {
     let webView = WKWebView(frame: .zero, configuration: configuration)
     webView.navigationDelegate = context.coordinator
     #if os(OSX)
-    webView.setValue(false, forKey: "drawsBackground")
+    webView.setValue(false, forKey: "drawsTransparentBackground")
```
Comment (marshallino16) [inline Sources/CodeMirror-SwiftUI/Representables/CodeView.swift:127]: This is causing a white flick on macOS 10.15.6 for me. Can you check on your side please?

Comment (L1cardo) [reply Sources/CodeMirror-SwiftUI/Representables/CodeView.swift:127]: > This is causing a white flick on macOS 10.15.6 for me. Can you check on your side please?

Actually, I can not see any difference, but it seems `"drawsBackground"` is better.
https://developer.apple.com/forums/thread/121139?answerId=376655022#376655022

**Comment by L1cardo** [comment]
`drawsBackground = false`
before

`drawsTransparentBackground = true`
after

**Comment by L1cardo** [comment]
It seems there is no difference except that one is white background and the other is grey(default or clear)

**Pull request merged**